core/proto/keytransparency_v1_proto/keytransparency_v1_proto.proto

```diff
@@ -27,11 +27,29 @@ import "crypto/keyspb/keyspb.proto";
 import "crypto/sigpb/sigpb.proto";
 import "trillian.proto";
 import "trillian_map_api.proto";
+import "google/rpc/status.proto";
 
 //
 // Data types.
 //
 
+// User represents plain account information that gets committed to and obfuscated in Entry.
+message User {
+  // domain_id specifies the domain.
+  string domain_id = 1;
+  // app_id specifies that application.
+  string app_id = 2;
+  // user_id specifies the user.
+  string user_id = 3;
+  // public_key_data is the public key material for this account.
+  bytes public_key_data = 4;
+  // authorized_keys is the set of keys allowed to sign updates for this entry.
+  repeated keyspb.PublicKey authorized_keys = 5;
+  // status is set when account is part of a batch operation.
+  google.rpc.Status status = 6;
```

Comment:
Just out of curiosity, how is this status gonna be used? Can't we use the returned error to report the status?

Reply:
Typically, when you create or modify an object, you return the same object as a result.  This works great when creating or modifying a single `User`, but gets a little muddy when we're talking about a batch of users. How should we associate the particular status with the particular user?  Here we've just put it inside the User object. It will only be filled in the response to a batch message.

Reply:
SG, thanks for the explanation!